lms/envs/bok_choy_static.py

```diff
@@ -0,0 +1,7 @@
+"""
+Settings for building static assets for Bok Choy tests.
+"""
+
+# TODO: update the LMS pipeline for Bok Choy to use compiled assets
```

Comment:
I'm afraid I don't understand this part. Perhaps I'm missing something. I thought we'd continue to use compiled assets...it would just be compiled AND optimized.